src/XMakeBuildEngine/UnitTestsPublicOM/Definition/ProjectItem_Tests.cs

```diff
@@ -1441,6 +1441,45 @@ public void RemoveMetadataImported()
             }
            );
         }
+
+        //  TODO: Should remove tests go in project item tests, project item instance tests, or both?
```

Comment:
Ideally both. Practically, the common Evaluator bahaviour can go into ProjectItem_Tests and the differences introduced by ProjectInstance in ProjectItemInstance_Tests. Like whether to consider conditions or not.